### DISCUSSION TITLE
IE11 渲染 HTML 节点不支持

### DISCUSSION THREAD

**Discussion by wangyonghua** [antvis/X6#2154] [Q&A] [answered]
我们在IE11 使用渲染 HTML 节点，但是不支持，将你们官网demo
https://x6.antv.vision/zh/docs/tutorial/advanced/react#%E6%B8%B2%E6%9F%93-html-%E8%8A%82%E7%82%B9
也出现loading，无法正常展示在ie11，请问你们这块兼容IE11吗

**Comment by NewByVector** [comment, marked as answer]
HTML 节点使用 foreignObject 标签来渲染，IE11 一些版本不支持 foreignObject 标签。

**Reply by wangyonghua** [reply]
请问有什么推荐的办法解决类？

**Reply by NewByVector** [reply]
使用 SVG  节点。

**Comment by lloydzhou** [comment]
@wangyonghua 
刚刚收到issue那边关闭的通知，重新看了看这个问题。

我中间实现过一个与官方有点区别的注册节点的库https://github.com/lloydzhou/x6-html-shape 
默认使用html来渲染节点，不依赖foreignObject

缺点就是，由于挂载自定义节点的dom元素在svg上层，所以，一些svg实现的tool会被节点遮住，并且连线也会被遮住。
